packages/client/README.md

```diff
@@ -689,20 +766,35 @@ The library allows the user to choose a _partition key_, which simplifies publis
 The partition key can be given as an argument to the `publish` methods, and the library assigns a deterministic partition number automatically:
 
 ```js
-await client.publish('my-stream-id', msg, Date.now(), msg.vehicleId)
-
-// or, equivalently
+// msg.vehicleId being the partition key
+await streamr.publish(streamId, msg, Date.now(), msg.vehicleId)
+// or, equivalently, using the stream object
 await stream.publish(msg, Date.now(), msg.vehicleId)
 ```
+You can also specify the partition number as the last parameter:
+```js 
+await streamr.publish(streamId, msg, Date.now(), 4)
+// or, equivalently, using the stream object
+await stream.publish(msg, Date.now(), 4)
```

Comment:
I'd remove this part. Let's not advertise this way of publishing to partition. The example after this is preferred.